src/EditorFeatures/CSharpTest/Diagnostics/HideBase/HideBaseTests.cs

```diff
@@ -93,5 +93,30 @@ class App : Application
     public new int Test;
 }");
         }
+
+        [WorkItem(18391, "Issue where 'new' keyword would be in the wrong order on 'const' fields.")]
```

Comment:
:bulb: The "description" should actually be a link to the issue:

```
[WorkItem(18391, "https://github.com/dotnet/roslyn/issues/18391")]
```

With that said, what you did here makes complete sense. I'll update `WorkItemAttribute` as a separate pull request so this is clear to future contributors. 👍